### DISCUSSION TITLE
Welcome to DevelopersStack-OpenSourceNexus Discussions!

### DISCUSSION THREAD

**Discussion by Shawnarun** [Shawnarun/DevelopersStack-OpenSourceNexus#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Keerthika04** [comment]
Hiii,
I'm Keerthika, who is deeply passionate about coding. My journey began with small Visual Basic programs, sparking a desire to dive deeper into the world of coding and explore new things. What truly captivates me about coding is problem-solving, it brings me immense joy and a sense of fulfillment, akin to solving a puzzle. This is how I found my love for coding, and I'm particularly drawn to learning more about AI.

**Comment by Shanth05** [comment]
Hello,
I'm Pirashanth, following Bachelor of Information Communication Technology at Rajarata University of Sri Lanka. 
I know about HTML, CSS, JAVASCRIPT, REACT and now further learning about Backend. I am new to the GitHub community. My main goal is to make my skillset on a professional level. I am excited to contribtue to open-source projects.

**Comment by musaraf0101** [comment]
Hi,
I'm Musaraf, currently following Higher National Diploma In IT at SLIATE.
I know about HTML,CSS,JAVA,UI Design and now learning WordPress.
